engine/src/main/java/org/terasology/engine/core/module/ExternalApiWhitelist.java

```diff
@@ -109,6 +109,21 @@
             .add("org.lwjgl")
             .add("org.terasology.jnlua")
             .add("org.joml")
+            .add("io.reactivex.rxjava3.annotations")
+            .add("io.reactivex.rxjava3.core")
+            .add("io.reactivex.rxjava3.disposables")
+            .add("io.reactivex.rxjava3.exceptions")
+            .add("io.reactivex.rxjava3.flowables")
+            .add("io.reactivex.rxjava3.functions")
+            .add("io.reactivex.rxjava3.internal")
+            .add("io.reactivex.rxjava3.observables")
+            .add("io.reactivex.rxjava3.observers")
+            .add("io.reactivex.rxjava3.parallel")
+            .add("io.reactivex.rxjava3.plugins")
+            .add("io.reactivex.rxjava3.processors")
+            .add("io.reactivex.rxjava3.schedulers")
+            .add("io.reactivex.rxjava3.subjects")
+            .add("io.reactivex.rxjava3.subscribers")
```

Comment:
How did you decide what to add here, and what is not added (assuming that there must be something we leave out, or otherwise we could have used a single entry `io.reactivex.rxjava3`).

Reply:
This particular diff has been changed to replace the rxjava packages with reactor packages, but the same question still applies.